tests/xtro-sharpie/api-annotations-dotnet/macOS-BackgroundAssets.todo

```diff
@@ -0,0 +1,30 @@
+!missing-enum! BADownloadState not bound
```

Comment:
So we are adding a macOS todo yet removing the binding that were there, why is that? Can't we just simply add it to mac os x and remove the todo or clean the todo accordingly?